lldb/source/Plugins/SymbolLocator/Debuginfod/SymbolLocatorDebuginfodProperties.td

```diff
@@ -1,7 +1,13 @@
 include "../../../../include/lldb/Core/PropertiesBase.td"
 
 let Definition = "symbollocatordebuginfod" in {
-  def ServerURLs : Property<"server_urls", "Array">,
+  def ServerURLs : Property<"server-urls", "Array">,
     ElementType<"String">,
     Desc<"An ordered list of Debuginfod server URLs to query for symbols. This defaults to the contents of the DEBUGINFOD_URLS environment variable.">;
+  def SymbolCachePath: Property<"cache-path", "String">,
+    DefaultStringValue<"">,
+    Desc<"The path where symbol files should be cached. This defaults to LLDB's system cache location.">;
+  def Timeout : Property<"timeout", "UInt64">,
+    DefaultUnsignedValue<0>,
+    Desc<"Timeout (in seconds) for requests made to a DEBUGINFOD server. A value of zero defaults to DEBUGINFOD_TIMEOUT environment variable (or 90 seconds).">;
```

Comment:
I think this is rather weird: why not make 90 the default? If I specify zero I'd expect no timeout.

Reply:
Both of these settings exist to let lldb users override values that are configurable through environment variables used in the Debuginfod library. 0 means "use the DEBUGINFOD_TIMEOUT value" which then defaults to a timeout of 90 seconds.

What does "no timeout" mean? Like, just wait forever? I'm not sure that's an option in curl... I could switch the default value to -1, just to be more clearly a nonsensical value. @clayborg also seemed confused by this option without more explanation, so I think it needs a better description. How about this:

Timeout (in seconds) for requests made to a DEBUGINFOD server. A value of zero [could be -1 instead] indicates that the system default should be used, which is separately configurable through the DEBUGINFOD_TIMEOUT environment variable.

Reply:
Gotcha, that makes sense and that's not how I understood it when I read the description. Not sure if this is better but maybe something along the lines of: 

> Timeout (in seconds) for requests made to a DEBUGINFOD server. A value of zero means we use the debuginfod default timeout: DEBUGINFOD_TIMEOUT if the environment variable is set and 90 seconds otherwise.

Reply:
I make no claims to my ability to write readable prose. Your description makes at least much as sense as mine, so I'll just swipe it directly.